bogenliga/bogenliga-business-logic/src/main/java/de/bogenliga/application/business/passe/impl/dao/PasseDAO.java

```diff
@@ -169,6 +169,13 @@ public PasseDAO(final BasicDAO basicDao) {
             .andEquals(PASSE_TABLE_MATCH_ID)
             .compose().toString();
 
+    private static final String FIND_BY_WETTKAMPF_ID_AND_MITGLIED_ID = new QueryBuilder()
+            .selectAll()
```

Comment:
Gut, in anderen DAOs habt ihr aber auch noch klassisch die SQL-Strings erstellt. Hier mehr Absprache im Team, um die Homogenität -> Wartbarkeit zu erhöhen.